src/layouts/Resources.jsx

```diff
@@ -1,378 +1,166 @@
-        withCredentials: true,
-      });
+      await axios.post(`${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/resource-room`, params);
     } catch (err) {
       console.log(err);
     }
   }
 
-  render() {
-    const {
-      resourceCategories,
-      resourcePages,
-      categoryModalIsActive,
-      settingsIsActive,
-      selectedResourcePage,
-      resourceRoomName,
-      newResourceRoomName,
-      isSelectingFile,
-      isFileStagedForUpload,
-      stagedFileDetails,
-    } = this.state;
-    const { match, location } = this.props;
-    const { siteName } = match.params;
-    return (
-      <>
-        {/* Resource Room exists */}
-        <Header />
-        {/* main bottom section */}
-        <div className={elementStyles.wrapper}>
-          <Sidebar siteName={siteName} currPath={location.pathname} />
-          { settingsIsActive
-            ? (
-              <ComponentSettingsModal
-                modalTitle={"Resource Settings"}
-                siteName={siteName}
-                isNewFile={selectedResourcePage.isNewPost}
-                category={selectedResourcePage.category}
-                fileName={selectedResourcePage.fileName}
-                settingsToggle={this.settingsToggle}
-                type="resource"
-                pageFilenames={_.chain(resourcePages)
-                  .filter({ category: selectedResourcePage.category })
-                  .map((file) => file.fileName)
-                  .value()}
-              />
-            )
-            : null}
-
-          {/* main section starts here */}
-          <div className={contentStyles.mainSection}>
-            <div className={contentStyles.sectionHeader}>
-              <h1 className={contentStyles.sectionTitle}>Resources</h1>
-              { resourceRoomName
-                ? (
-                  <ResourceCategoryModal
+  return (
+    <>
+      <Header />
+      {/* main bottom section */}
+      <div className={elementStyles.wrapper}>
+        <Sidebar siteName={siteName} currPath={location.pathname} />
+        {/* main section starts here */}
+        <div className={contentStyles.mainSection}>
+          {/* Page title */}
+          <div className={contentStyles.sectionHeader}>
+            <h1 className={contentStyles.sectionTitle}>Resources</h1>
+          </div>
+          {
+            isLoading ? 'Loading Resources...'
+            :
+              resourceRoomName
+              ? <>
+                  {/* Category title */}
+                  <div className={contentStyles.segment}>
+                    Resource Categories
+                  </div>
+                  {/* Info segment */}
+                  <div className={contentStyles.segment}>
+                    <i className="bx bx-sm bx-info-circle text-dark" />
+                    <span><strong className="ml-1">Note:</strong> Categories cannot be empty, create a resource first to create a Category.</span>
```

Comment:
This should only be displayed if a resource room exists

Reply:
This is nested within the `resourceRoomName` check, so it only displays if a resource room exists